packages/web-components/src/components/rux-modal/rux-modal.tsx

```diff
@@ -73,26 +92,27 @@ export class RuxModal {
     handleClick(ev: MouseEvent) {
         const wrapper = this._getWrapper()
         if (ev.composedPath()[0] === wrapper) {
-            this.ruxModalClosed.emit(false)
+            // this.ruxModalClosed.emit(false)
             this.open = false
         }
     }
 
     @Watch('open')
-    validateName(isOpen: boolean) {
-        if (isOpen) {
+    handleOpen(isOpen: boolean) {
+        //? This is preventing the default focusing of a button if the footer slot exists.
+        //? I thought about runninng it anyway, but it could lead to cases where an unintentional button gets focused.
+        if (isOpen && !this.hasFooter) {
             setTimeout(() => {
                 const button = this._getDefaultButton()
-                button && button.focus()
-            })
+                if (button) {
+                    button.focus()
+                }
+            }, 0)
         }
+        this.open ? this.ruxModalOpened.emit() : this.ruxModalClosed.emit()
     }
 
-    private _handleModalChoice(e: MouseEvent) {
-        // convert string value to boolean
```

Comment:
All this method did was emit the ruxmodalclosed event with a true/false value. Now that we have an opened event, this is unnecessary because if closed is emitted, the modal closed.